cmd/setup/43.sql

```diff
@@ -0,0 +1,5 @@
+CREATE INDEX CONCURRENTLY events2_current_sequence2
+    ON eventstore.events2 USING btree
+    (aggregate_id ASC, aggregate_type ASC, instance_id ASC, sequence DESC);
+
+DROP INDEX eventstore.events2_current_sequence;
```

Comment:
You sure this works? IFAIK you can't `CREATE INDEX CONCURRENTLY` in a transaction. `CREATE` and `DROP` need to be in separate files so that postgres doesn't create an implicit transaction.

Reply:
As i suspected, pipeline fails: https://github.com/zitadel/zitadel/actions/runs/12649347901/job/35245721931?pr=9142#step:8:98

Reply:
you are right